### ISSUE TITLE
Provide additional alt-text for control interface icon elements

### ISSUE THREAD

**Issue by anathemamask** [foundryvtt/foundryvtt#5386]
A number of control elements listed below are missing alt-text which is required by WCAG to provide a cleaner interface for screen reading software.

- Macro bar buttons
- Macro page controls
- Sidebar navigation
- Scene navigation collapse and sidebar collapse controls
- Folder 'create folder' and 'create {document}' icon buttons
